[2025-05-22T14:21:23.514+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_stream_dag.fetch_weather scheduled__2025-05-22T13:00:00+00:00 [queued]>
[2025-05-22T14:21:23.526+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_stream_dag.fetch_weather scheduled__2025-05-22T13:00:00+00:00 [queued]>
[2025-05-22T14:21:23.526+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-22T14:21:23.544+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): fetch_weather> on 2025-05-22 13:00:00+00:00
[2025-05-22T14:21:23.549+0000] {standard_task_runner.py:60} INFO - Started process 179 to run task
[2025-05-22T14:21:23.553+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_stream_dag', 'fetch_weather', 'scheduled__2025-05-22T13:00:00+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/weather_stream_dag.py', '--cfg-path', '/tmp/tmpoe0pk8ca']
[2025-05-22T14:21:23.556+0000] {standard_task_runner.py:88} INFO - Job 29: Subtask fetch_weather
[2025-05-22T14:21:23.575+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-22T14:21:23.619+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_stream_dag.fetch_weather scheduled__2025-05-22T13:00:00+00:00 [running]> on host ac55c3aa9876
[2025-05-22T14:21:23.749+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_stream_dag' AIRFLOW_CTX_TASK_ID='fetch_weather' AIRFLOW_CTX_EXECUTION_DATE='2025-05-22T13:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-22T13:00:00+00:00'
[2025-05-22T14:21:23.923+0000] {logging_mixin.py:188} INFO - [✓] Données météo sauvegardées : data/raw/weather/weather_2025-05-22T14-21-23.json
[2025-05-22T14:21:23.926+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-22T14:21:23.943+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_stream_dag, task_id=fetch_weather, execution_date=20250522T130000, start_date=20250522T142123, end_date=20250522T142123
[2025-05-22T14:21:24.008+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-22T14:21:24.048+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
